src/data-access/MwApiLangCodeRetriever.ts

```diff
@@ -0,0 +1,21 @@
+import { MwApi } from '@/@types/mediawiki';
+import LangCodeRetriever from './LangCodeRetriever';
+import { processWbGetClaimsResponse, WbGetClaimsResponse } from './apiBaseLangCodeRetriever';
+
+export default class MwApiLangCodeRetriever implements LangCodeRetriever {
+
+	public constructor( private api: MwApi, private languageCodeProperty: string ) {
```

Comment:
Yep, Typescript magic. Not convinced by it myself yet, but I thought I try it out. Apparently they are called "parameter properties": https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties

Reply:
Yeah, but also I think I should actually assert that the languageCodeProperty is actually a string and not `null` because of a config error. So, I'll probably move to the default style again, at least for this case.

Reply:
Done.